### DISCUSSION TITLE
Setting paper size height on continuous rolls using Brother label printer and expo-print

### DISCUSSION THREAD

**Discussion by borkanee** [expo/expo#16999] [Archived: Questions]
Hi! I'm trying to print some labels using expo-print and a Brother label printer (QL 820NWB) but I have an issue with setting the height of the labels. For some reason it always prints out 80.4mm. Passing height and width as argument to the printAsync method does not change the paper size. Any ideas? Has anyone used expo-print with a label printer? Perhaps there are other libraries that can help with this? Thanks!

**Comment by andrew-bierman** [comment]
+1 I had a similar issue with that same printer model, any chance you found a solution for this?

**Comment by guiccbr** [comment]
For pdfs we were able to make it work reasonably well by just making sure the pdf is generated with the right width and height. The printer will then respect it.
